src/Extension/ValueResolvers/OptionValueResolver.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Extension\ValueResolvers;
+
+use SprykerSdk\Sdk\Core\Appplication\ValueResolver\AbstractValueResolver;
+use SprykerSdk\SdkContracts\Entity\ContextInterface;
+use SprykerSdk\SdkContracts\ValueResolver\ConfigurableValueResolverInterface;
+
+class OptionValueResolver extends AbstractValueResolver implements ConfigurableValueResolverInterface
```

Comment:
What do you think about extending it from StaticResolver? This way it can inherit all behavior except for value formatting, for example choice values.